roxctl/common/npguardutils/netpolutils.go

```diff
@@ -0,0 +1,36 @@
+package npguardutils
```

Comment:
This is not a good name as it does not reflect why this package exists and the usage of exported functions  is a bit ugly.

I would suggest to revert that commit and think that through again. Let me try to comment on some code examples to demonstrate why I think this should be renamed again.

Reply:
This package should be considered internal. This should be decided after we agree on the contents of this package.